src/tomography/tensornetwork-processtomography.jl

```diff
@@ -538,6 +538,7 @@ function gradients(
   data::Matrix{Pair{String,Pair{String,Int}}};
   sqrt_localnorms=nothing,
   trace_preserving_regularizer=nothing,
+  kwargs...
```

Comment:
Is this `kwargs` needed?

Reply:
Yeah this should not be needed in principle, will double check.